app/models/space.rb

```diff
@@ -45,6 +45,9 @@ class Space < ApplicationRecord
   # @returns {ActiveRecord::Relation<Utilities>}
   has_many :utilities, inverse_of: :space, dependent: :destroy
 
+  attribute :brand_header, :boolean
+  attribute :brand_color, :string
```

Comment:
Comments from group code review: it might be nice to use more specific names here, like `header_bg_color`. Feels like "brand_color" would imply a lot more than what we currently plan to do with this color.